### PR TITLE
chore(config): move all former components in a old repo + rm storybook

### PR DESCRIPTION
**Pull request by cnairi**
## DESCRIPTION

We moved all the styled components to an old folder. 
Remove storybook to use our next website.

## COMPATIBILITY

- [ ] Tested on Safari (desktop)
- [ ] Tested on Chrome (desktop)
- [ ] Tested on Firefox (desktop)
- [ ] Tested on mobile device sizes
- [ ] Tested on tablet device sizes
- [ ] Tested on IOS Safari (either device or simulator)

## QA

- [ ] Thoroughly tested in local environment
- [ ] Added tests for all new features
- [ ] Added tests that considered edge cases

### CODE REVIEW

File: lib/src/components/Button/docs/examples/icons.tsx [hunk 1]
```diff
@@ -1,23 +1,22 @@
-import { Button } from '@/Button'
-import { WttjIcon } from '@/Icons'
+import { Button } from '../../index'
 
 const Example = () => {
   return (
     <>
       <Button>
-        <WttjIcon />
+        <i className="wui-icon-add wui-icon-font" data-wui-icon-font />
```
Comment (P1X3L) [inline lib/src/components/Button/docs/examples/icons.tsx]: we can get rid of `data-wui-icon-font` and rely on `.wui-icon-font` and change the selector here (https://github.com/WTTJ/welcome-ui/pull/2813/files#diff-ec6f34fab33c004f73ff41ee152f2a8d61fad915d738fbfe604318ff9d6ebde6R57-R58)

Comment (P1X3L) [reply lib/src/components/Button/docs/examples/icons.tsx]: I added data attribute because there was a tailwind limitation

File: lib/src/components/Button/docs/examples/link.tsx [hunk 2]
```diff
@@ -9,7 +8,7 @@ const Example = () => {
       rel="noopener nofollow"
       target="_blank"
     >
-      <WttjIcon />
+      <i className="wui-icon-add wui-icon-font" data-wui-icon-font />
```
Comment (P1X3L) [inline lib/src/components/Button/docs/examples/link.tsx:11]:
```suggestion
      <i className="wui-icon-add wui-icon-font" />
```

File: lib/src/components/Button/button.module.scss [hunk 3]
```diff
@@ -53,17 +53,11 @@
...
 
-  /* This is a workaround because the hover peudo state is not compatible with Tailwind 4 */
-  /* https://github.com/storybookjs/storybook-addon-pseudo-states/issues/140 */
-  &.pseudo-hover {
-    background-color: var(--backgroundColorHover);
-    border-color: var(--borderColorHover);
-  }
-
   /* CHILDREN */
   i[data-wui-icon-font],
```
Comment (P1X3L) [inline lib/src/components/Button/button.module.scss]:
```suggestion
  i.wui-icon-font,
```

File: lib/src/components/Button/button.module.scss [hunk 4]
```diff
@@ -53,17 +53,11 @@
...
   svg[data-wui-icon] {
```
Comment (P1X3L) [inline lib/src/components/Button/button.module.scss]:
```suggestion
  svg.wui-icon {
```

File: stylelint.config.mjs [hunk 5]
```diff
@@ -2,7 +2,7 @@ export default {
   extends: ['stylelint-config-recommended'],
   overrides: [
     {
-      files: ['**/src/tailwind**/**/*.{css,scss}'],
+      files: ['**/src/components/**/*.{css,scss}', '**/src/theme/**/*.{css,scss}'],
```
Comment (P1X3L) [inline stylelint.config.mjs]:
```suggestion
      files: ['**/src/**/*.{css,scss}'],
```
Since new version is the only one dealing with `.css` and `.scss` files, it should work all the same

File: website/app/components/[id]/layout.tsx [hunk 6]
```diff
@@ -47,7 +48,7 @@ const Layout = async ({ children, params }: LayoutProps) => {
       <Sidebar display={{ _: 'none', lg: 'flex' }} isSubPage menu={pages} />
       <div>
         <Flex direction="column" gap="xl" mb="lg">
-          <Text mt="3xl" variant="h1">
+          <Text className="mt-128" mt="3xl" variant="h1">
```
Comment (theo-mesnil) [inline website/app/components/[id]/layout.tsx:51]:
```suggestion
          <Text mt="3xl" variant="h1">
```

File: website/app/page.tsx [hunk 7]
```diff
@@ -26,7 +27,7 @@ const Home = () => {
           Welcome UI is a customizable design system library made with react, typescript,
           styled-components, ariakit and a lot of love 💛
         </Text>
-        <Button as={NextLink} href="/foundations/getting-started" mt="3xl" size="lg">
+        <Button as={NextLink} className="!mt-3xl" href="/foundations/getting-started" size="lg">
```
Comment (theo-mesnil) [inline website/app/page.tsx:30]: I know this not working without `!` but we need to fix after button instead of add important

```suggestion
        <Button as={NextLink} className="mt-3xl" href="/foundations/getting-started" size="lg">
```

File: website/app/page.tsx [hunk 8]
```diff
@@ -60,8 +61,8 @@ const Home = () => {
             </Text>
             <Button
               as="a"
+              className="!mt-3xl"
```
Comment (theo-mesnil) [inline website/app/page.tsx:64]:
```suggestion
              className="mt-3xl"
```

File: website/app/page.tsx [hunk 9]
```diff
@@ -137,8 +138,8 @@ const Home = () => {
           </Button>
           <Button
             as="a"
+            className="!mt-0 !xs:md-md"
```
Comment (theo-mesnil) [inline website/app/page.tsx:141]:
```suggestion
            className="mt-md md:mt-0"
```

**Review by guillaumewttj — APPROVED** [review]

File: stylelint.config.mjs (hunk 5 above)
Comment (cnairi) [reply stylelint.config.mjs]: done

File: lib/src/components/Button/button.module.scss [hunk 10]
```diff
@@ -54,19 +54,12 @@
   }
 
   /* CHILDREN */
-  i.wui-icon-font,
-  svg.wui-icon {
+
+  :global(i.wui-icon-font),
```
Comment (Leiksa) [inline lib/src/components/Button/button.module.scss:59]: ⚠️ About using `:global` for icons
We might be tempted to use `:global(i.wui-icon-font)` or `:global(svg.wui-icon)` to style icons
However, `:global` makes the styles **non-scoped**, meaning that all matching icons across the project (outside this button) will inherit these styles, which can break their appearance elsewhere.

Using `data-wui-icon` is safer and ensures styles remain scoped. 🤔 
WDYT @P1X3L @cnairi @guillaumewttj ?

Comment (P1X3L) [reply lib/src/components/Button/button.module.scss:59]: Are you sure about the non scoped thing?

If I take a look at the compiled selector, it looks like this -> 
```
.button_root__1_npZ:has(i.wui-icon-font), .button_root__1_npZ:has(svg.wui-icon) {
    padding-inline: var(--paddingInlineHasIcon);
}
```

It looks scoped and safe to me 🤔 
Wdyt @Leiksa? Am I missing something?

Comment (cnairi) [reply lib/src/components/Button/button.module.scss:59]: I created a slack thread here: https://wttj.slack.com/archives/C09CT2BSF0B/p1757935562409669 to discuss this. I'll leave this discussion unresolved for now 😄

Comment (Leiksa) [reply lib/src/components/Button/button.module.scss:59]: You are completely right @P1X3L !
Sorry this is my bad I didn't know about nesting :global selector 🙏🏻
Thank you @cnairi for the doc shared in slack!!
